www/proxy-sso/+POST_DEINSTALL.post

```diff
@@ -0,0 +1,5 @@
+rm /usr/local/etc/squid/pre-auth/20-negotiate.auth.conf
+if [ -f /var/run/squid/squid.pid ]; then
+	service squid reload
```

Comment:
I know what it's trying to achieve, but it's tricky as it doesn't work on a stock FreeBSD, it would kill the service. In OPNsense it works, maybe we need some type of reload command embedding in the package Makefile to signal reverse dependencies of base plugins?

PLUGIN_USES=     squid

pointing to the squid.inc plugin file and doing a reload if possible?

Reply:
in any case, a configd call would be far safer here, preferably in the background